### PR TITLE
ProceduralProbes to mirror

### PR DESCRIPTION
**Pull request by politas**
CC-BY-SA-3.0 licence

### CODE REVIEW

**Comment by Dazpoet** [comment]
Jenkins is being Jenkins while doing Jenkins things.

**Comment by techman83** [comment]
Actually this one failed validation. I'll take a closer look.

```
Traceback (most recent call last):
  File "./ckan-validate.py", line 53, in <module>
    main()
  File "./ckan-validate.py", line 42, in main
    validate(json.load(ckan_file), schema)
  File "/usr/lib/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting : delimiter: line 11 column 5 (char 389)
```

**Comment by Dazpoet** [comment]
That's a big oops on my part then, sorry about that and I'll make sure to read up better next time :s

File: ProceduralProbes/ProceduralProbes-1.ckan [hunk 1]
```diff
@@ -6,8 +6,8 @@
     "author": "JarDev",
     "license": "CC-BY-SA-3.0",
     "resources": {
-        "kerbalstuff": "https://kerbalstuff.com/mod/1068/[1.0.4]ProceduralProbes",
-        "x_screenshot": "https://kerbalstuff.com/content/JarDev_10353/1.0.4ProceduralProbes/1.0.4ProceduralProbes-1439209930.2996624.png"
+        "homepage": "http://felicetti-a.eu/",
+        "http://ckan1.spacedock.info/storage/JarDev_10353/1.0.4ProceduralProbes/1.0.4ProceduralProbes-1439209930.2996624.png"
```
Comment (techman83) [inline ProceduralProbes/ProceduralProbes-1.ckan]: Missing
```
 "x_screenshot": 
```
I'll submit a PR with the addition.

**Pull request merged**